From 224b6067588de113dfb9d5c70aedd8e59024c8ea Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:29 -0400
Subject: [usb] pci-quirks: amd_chipset_sb_type_init() can be static

Message-id: <1410887463-20674-18-git-send-email-dzickus@redhat.com>
Patchwork-id: 95103
O-Subject: [RHEL7 PATCH 017/591] usb: pci-quirks: amd_chipset_sb_type_init() can be static
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 40b3dc6da05c4ac0e317723a22eaa807c4b98648
Author: Fengguang Wu <fengguang.wu@intel.com>
Date:   Thu Sep 26 11:56:44 2013 -0700

    usb: pci-quirks: amd_chipset_sb_type_init() can be static

    Acked-by: Huang Rui <ray.huang@amd.com>
    Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/pci-quirks.c b/drivers/usb/host/pci-quirks.c
index 367ab42..443b61d 100644
--- a/drivers/usb/host/pci-quirks.c
+++ b/drivers/usb/host/pci-quirks.c
@@ -118,7 +118,7 @@ static DEFINE_SPINLOCK(amd_lock);
  *
  * Returns: 1 if it is an AMD chipset, 0 otherwise.
  */
-int amd_chipset_sb_type_init(struct amd_chipset_info *pinfo)
+static int amd_chipset_sb_type_init(struct amd_chipset_info *pinfo)
 {
  u8 rev = 0;
  pinfo->sb_type.gen = AMD_CHIPSET_UNKNOWN;
-- 
1.7.1